recordset/recordset.js

```diff
@@ -219,6 +224,7 @@
                 $log.warn(exception);
```

Comment:
I think the whole catch should be replaced by `catchAll` (assuming that `catchAll` will eventually pop-up the error if it cannot handle it)